packages/babel-preset-env/src/debug.ts

```diff
@@ -13,6 +16,18 @@ export const logPlugin = (
 
   const support = list[item];
 
+  // TODO(Babel 8): Remove this. It's needed to keep outputting proposal-
+  // in the debug log.
```

Comment:
The changes in this file are to continue printing `proposal-` instead of `transform-` in the debug log. I think that it would be ok to print `transform-`, since the debug output is just logging and thus I wouldn't consider it a breaking change. What do you think?